### DISCUSSION TITLE
service

### DISCUSSION THREAD

**Discussion by ENI1999** [Chia-Network/chia-blockchain#18562] [Support]
new version Downloads  Chia 2.4.1 installs ok but starting service just lauding for all day not starting

**Comment by BrandtH22** [comment]
Hey @ENI1999 , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

- stop chia
- if you have not already, please install the latest version from chia.net/downloads/
- rename the `.chia` folder to `.chia_old`
- open command prompt and run `chia init` (this will create a new `.chia` folder)
- move the old DB to the new location (move `~/.chia_old/mainnet/db` to `~/.chia/mainnet/db`)
- start chia (it may take upwards of 30 minutes to start the first time)

Note - this generates a new config file so you will want to verify your settings for:
- plots: readd plots via the gui or by manually editing the config (always make sure chia is stopped before editing the config)
- plotNFTs: readd these by resyncing the wallet with the gui settings / advanced / resync (or by deleting the wallet db with chia stopped)
- payout addresses: confirm your payout addresses set on the farm tab of the GUI
- any other custom settings

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
